services/web/server/src/simcore_service_webserver/folders/_folders_api.py

```diff
@@ -273,7 +273,7 @@ async def delete_folder(
         )
         workspace_is_private = False
 
-    # Check user has acces to the folder
+    # Check user has access to the folder
```

Comment:
not sure I get the sense of this comment now. aren't you checking on line 267?

Reply:
No, above I check whether the user has access to the workspace. Based on that I know the context and then I check the access to the folder via that context. (Noted for improvement)